src/solc_0.8/polygon/child/ChildLandBaseToken.sol

```diff
@@ -0,0 +1,85 @@
+//SPDX-License-Identifier: MIT
+/* solhint-disable func-order, code-complexity */
+pragma solidity 0.8.2;
+
+import "../../common/BaseWithStorage/ERC721BaseToken.sol";
+
+contract ChildLandBaseToken is ERC721BaseToken {
+    // Our grid is 408 x 408 lands
+    uint256 internal constant GRID_SIZE = 408;
+    // @review DO we need this?
+    uint256 internal constant LAYER = 0xFF00000000000000000000000000000000000000000000000000000000000000;
+
+    mapping(address => bool) internal _minters;
+    event Minter(address superOperator, bool enabled);
+
+    constructor(
+        address trustedForwarder,
+        uint8 chainIndex,
+        address admin
+    ) {
+        _admin = admin;
+        ERC721BaseToken.__ERC721BaseToken_initialize(chainIndex);
+        ERC2771Handler.__ERC2771Handler_initialize(trustedForwarder);
+    }
+
+    /// @notice Enable or disable the ability of `minter` to mint tokens
+    /// @param minter address that will be given/removed minter right.
+    /// @param enabled set whether the minter is enabled or disabled.
+    function setMinter(address minter, bool enabled) external {
```

Comment:
I would remove that, the only minter will be Matic bridge

Reply:
Actually... will there be LandSale contracts deployed on Polygon that would need to be able to mint?
cc @wighawag